rollup.config.js

```diff
@@ -36,7 +36,7 @@ export default {
   entry: 'src/index.js',
   targets,
   exports: esm ? 'named' : 'default',
-  moduleName: 'Downshift',
+  moduleName: 'ReactToggled',
```

Comment:
Ha! You caught me! This repo was a copy/paste from downshift.

Thanks for the contrib!